sdk/servicebus/service-bus/src/connectionContext.ts

```diff
@@ -45,13 +46,13 @@ export namespace ConnectionContext {
 
   export function create(
     config: ConnectionConfig,
-    tokenProvider: TokenProvider,
+    tokenCredential: SharedKeyCredential | TokenCredential,
```

Comment:
The [connectionContext.ts file in the feature/service-bus-track-2](https://github.com/Azure/azure-sdk-for-js/blob/feature/service-bus-track-2/sdk/servicebus/service-bus/src/connectionContext.ts#L49) branch already has these changes, so I am not sure why these are appearing in the diffs as part of this PR.

Reply:
I believe the upstream branch was updated after the PR was created, it should have still pulled in the updates..
Will open a new PR